### PR TITLE
added new trainings

### PR DESCRIPTION
**Pull request by sefasaid**

### CODE REVIEW

File: 1-html-portfolio/Assignment 1.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Assignment 1
+
+The web page doesn't contain anything yet. Let's get started 💻!
+
+Start with implementing your image and your name:
+>
+> - `Create a division (div element)`:
+>   - Inside this division add following items
+> - `Add your image (img element)`:
+>   - Add your image with img tag and url. Please add alt tag to you image element
```
Comment (FrankMerema) [inline 1-html-portfolio/Assignment 1.md]:
```suggestion
>   - Add your image with img tag and url. Please add an alt tag to you image element
```

File: 1-html-portfolio/Assignment 2.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Assignment 2
+
+Now let's add your hobbies!
+
+To do that please follow instructions:
```
Comment (FrankMerema) [inline 1-html-portfolio/Assignment 2.md]:
```suggestion
To make it work, please follow these instructions:
```

File: 1-html-portfolio/Assignment 3.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Assignment 3
+
+Time to add connecting links!
+
+To do that please follow instructions:
```
Comment (FrankMerema) [inline 1-html-portfolio/Assignment 3.md]: Same as above comment

File: 2-css/examples/index.html [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+            width: 200px;
+            height: 200px;
+        }
+
+    </style>
+</head>
+
+<body>
+    <div class="portfolio">
+        <img class="user-image" src="https://upload.wikimedia.org/wikipedia/commons/7/71/2010-kodiak-bear-1.jpg">
```
Comment (FrankMerema) [inline 2-css/examples/index.html:32]: The image of the bear looks stretched, could you apply some css to center it more nicely?

File: 1-html-portfolio/Assignment 1.md (hunk 1 above)
Comment (sefasaid) [reply 1-html-portfolio/Assignment 1.md]: fixed

File: 2-css/examples/index.html (hunk 4 above)
Comment (sefasaid) [reply 2-css/examples/index.html:32]: added object-fit: cover; looks better now

File: 1-html-portfolio/Assignment 3.md (hunk 3 above)
Comment (sefasaid) [reply 1-html-portfolio/Assignment 3.md]: fixed

File: 1-html-portfolio/Assignment 2.md (hunk 2 above)
Comment (sefasaid) [reply 1-html-portfolio/Assignment 2.md]: Fixed

**Review by FrankMerema — APPROVED** [review]

**Pull request merged**